### ISSUE TITLE
Evernote limit basic accounts

### ISSUE THREAD

**Issue by artmg** [baumgarr/nixnote2#173]
Hi @baumgarr

I have really appreciated the hard work you have put into evolving Evernote clients over the years, but now that [Evernote are limiting Basic (free) accounts to ONLY TWO devices](https://blog.evernote.com/blog/2016/06/28/changes-to-evernotes-pricing-plans) I wonder if you still expect to continue development?

Personally I'm disappointed the company can't produce a client for a common OS but still wants to charge users nonetheless, and I'm wondering if I should not be looking for an alternative service.

I'd really appreciate to hear your view
Thanks
Art

**Comment by baumgarr** [comment]
Yes. I still plan on it. I have the next release built and will hopefully upload it tonight or tomorrow. I don't have any idea how the two client limit will impact things. My guess is that if there are more than two devices you'll get a sync error.

**Comment by artmg** [comment]
cool - I look forward to beta 8, and hopefully it should ripple out into the daily builds PPA smoothly

Thanks for letting me know that you'll still be behind the project. Cheers, Art

**Comment by Chemrat** [comment]
At least for now, API requests do not count towards device limit, see https://help.evernote.com/hc/en-us/articles/218558068-Devices-FAQ

> Other Evernote products, and third-party apps that use the Evernote API integration, aren’t affected by device limits. Examples: Evernote Web, Evernote on Apple Watch, Evernote for Windows Touch, Web Clipper, Skitch, and Penultimate.

**Comment by artmg** [comment]
Thanks for that link @Chemrat - encouraged by this I have also reviewed other articles on the recent changes to price plans and it appears that the limit is 2 devices with "Evernote Clients installed and logged in". All current responses seem to indicate that web access and API access (like through nixnote2) DO NOT count towards the device limit.

I was also concerned because I got a second email saying "**You are currently over this limit**", but when I went into Evernote / **Account** / **Devices** I found it was due to old clients on devices I no longer used. I simply clicked "**revoke access**" next to those and am now within the two device limit.
